javascript/emscripten.dockerfile

```diff
@@ -1,10 +1,34 @@
-FROM emscripten/emsdk AS build_image
+FROM emscripten/emsdk AS base_image
 
 RUN apt-get -qq -y update \
     && DEBIAN_FRONTEND="noninteractive" \
        apt-get -qq install -y --no-install-recommends \
             libmsgpack-dev
 
-FROM emscripten/emsdk AS final_image
-COPY --from=build_image /usr/include/msgpack.* /msgpack/
-COPY --from=build_image /usr/include/msgpack/  /msgpack/msgpack/
+FROM emscripten/emsdk AS build_image
```

Comment:
With introduction of nice scratch don't see a point in separating two images (as both are emsdk-based anyway), but who cares...

Reply:
Yes it is unesscay now, was initially added as I was planning to add extra steps to the build_image